bindings/python/tests/compiler_tf_test.py

```diff
@@ -0,0 +1,82 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import sys
+import tempfile
+import unittest
+
+from pyiree.compiler2.tf import *
+
+if not is_available():
+  print(f"Skipping test {__file__} because TensorFlow is not installed")
+  sys.exit(0)
+
+import tensorflow as tf
+
+
+class SimpleArithmeticModule(tf.Module):
+
+  @tf.function(input_signature=[
+      tf.TensorSpec([4], tf.float32),
+      tf.TensorSpec([4], tf.float32)
+  ])
+  def simple_mul(self, a, b):
+    return a * b
+
+  @tf.function(input_signature=[
+      tf.TensorSpec([128, 3072], tf.float32),
+      tf.TensorSpec([3072, 256], tf.float32),
+  ])
+  def simple_matmul(self, a, b):
+    return tf.matmul(a, b)
+
+
+class TfCompilerTest(unittest.TestCase):
+
+  def testImportSavedModel(self):
+    import_mlir = compile_saved_model(self.smdir,
+                                      import_only=True).decode("utf-8")
+    self.assertIn("func @simple_matmul", import_mlir)
+
+  def testCompileSavedModel(self):
```

Comment:
Worth mirroring this to have a coherence check on the SigDef side? (I'm assuming `tf` isn't amicable to generating both types in the same runtime). Realistically we'll have models that test that too.

Reply:
I'll add a todo. I legitimately don't know how to generate one of those reliably and likely need another test file for it once I figure out the incantation.